setup.py

```diff
@@ -46,7 +46,7 @@
     author_email = AUTHOR_EMAIL,
     maintainer = __author__,
     maintainer_email = AUTHOR_EMAIL,
-    license = '3-clause BSD',
+    license = 'BSD-3-Clause',
```

Comment:
This is already updated in maser.

Reply:
Must be some bad merge with another branch that I did. Just sent a commit with latest updates to settings, including a new test file.